### DISCUSSION TITLE
tag to filter the list of projects

### DISCUSSION THREAD

**Discussion by pcadeot** [aljawaid/TagManager#10] [Q&A] [answered]
Hi
I'm sorry I don't have the possibility to check it out by myself, so I'll ask here : 
I understand it helps managing tags used to characterize tasks on a global level (common to all projects) and also on a project specific level, but does the plugin adds a way to tag a project in itself, in the way you can filter the list of projects, say on a specific topic ?
To explain it a little more, if needed : I am in an organization with several teams.
Kanboard is used by the whole organization.
Does this plugin could help me to filter in the list of projects for a specific team : I would like to set a tag at a project level, not used for tasks, but for project itself, a tag using the name of a team. So that from the main view, having a projects tags pull down list to filter the tag with the team name I'm looking for...
Thanks for your time,
Pierre

**Comment by aljawaid** [comment, marked as answer]
Hello, from what I understand...

- Assigning a tag to a project as a form of identification - I think the answer is no.
- You can create a tag per team and then filter by tag, yes...
- You could also look at the 'Group_assign' plugin to assign projects to groups. BUt then that will also have permissions to the groups also.

You can only search for projects, not filter them - this is the default behaviour.

**Reply by pcadeot** [reply]
Thanks I will look for that :)
I had the feeling that when you search in the main page, in fact you search **through tasks only** not projects names, that's a tad counter intuitive... did I miss something ?
![image](https://github.com/aljawaid/TagManager/assets/50657616/c2a8b814-7e94-4eb0-808b-6b4c43ee8e6b)
